### PR TITLE
`azurerm_web_application_firewall_policy` - support for the `excluded_rule_set ` property

### PR DESCRIPTION
**Pull request by ms-zhenhua**
image

### CODE REVIEW

File: internal/services/network/web_application_firewall_policy_resource.go [hunk 1]
```diff
@@ -195,6 +195,47 @@ func resourceWebApplicationFirewallPolicy() *pluginsdk.Resource {
 											string(network.OwaspCrsExclusionEntrySelectorMatchOperatorStartsWith),
 										}, false),
 									},
+									"exclusion_managed_rule_set": {
```
Comment (katbyte) [inline internal/services/network/web_application_firewall_policy_resource.go:198]: this is in a block called `managed_rules`, so should it maybe be
```suggestion
									"excluded_rule_set": {
```

so we get `managed_rules.excluded_rule_set`

**Review by katbyte — CHANGES_REQUESTED** [review]

**Review by katbyte — APPROVED** [review]
LGTM ⏳

**Comment by katbyte** [comment]
needs terrafmt to be run

**Comment by ms-zhenhua** [comment]
Hi @katbyte, thank you for your help. Now all checks have passed. Kindly help check it again.

**Pull request merged**